h/views/api_config.py

```diff
@@ -0,0 +1,92 @@
+import venusian
+
+from h.util import cors
+
+
+cors_policy = cors.policy(
+    allow_headers=(
+        'Authorization',
+        'Content-Type',
+        'X-Annotator-Auth-Token',
```

Comment:
I know this was already here, but `X-Annotator-Auth-Token` is unused by the client and it can be removed.

Reply:
Addressed separately in #4636.